covidapi/app.py

```diff
@@ -0,0 +1,3 @@
+from fastapi import FastAPI
+
+app = FastAPI()
```

Comment:
Yes! That's the idea. I would like to keep the API views in the subfolder (we can discuss the naming in the separate issue #5 ) and only import the routes like this:

```
from fastapi import FastAPI
from api.views import dailyreport_controller


app = FastAPI()
app.include_router(dailyreport_controller.router)
```

Reply:
p.s: just please add the missing line ad the end of file and we can start merging this.

Reply:
oops, missed this comment, I will raise another PR...